.vaunt/config.yaml

```diff
@@ -0,0 +1,89 @@
+version: 0.0.1
+achievements:
+    - achievement:
+          name: Docs Dash PR Submitted
+          icon: <icon_url>
+          description: Submit a PR with the label 2024-tidb-docs-dash between January 9th - 12th 2024!
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
```

Comment:
Can we set the range of hours a little wider or only keep the date condition? I am not sure if a PR submitted at 08:00:00 (China time) on 2024-01-09 will also be counted for this.

Reply:
@qiancai we're using UTC so if someone in China submits something at 08:00:00 local time won't count. I like to keep it fair and give people everywhere 72 hours. People in China will be able to contribute until the afternoon of the 12th.